### PR TITLE
Create Get Involved page

### LINKED ISSUE #1423: Create a "How to get involved in Jakarta EE" page
Please create a "How to get involved in Jakarta EE" page that outlines various methods for developers to get involved in and contribute to Jakarta EE. The mockup is here:

https://projects.invisionapp.com/share/AX10M9QL74RN#/screens/462733154

Please find the content for the Testimonials section here (we need to list them on rotation when the page is refreshed): 

https://docs.google.com/document/d/19A55nEm8mwQcZrGOSZqbNUModB9e5MVrybl_k0zWBsI/edit

We can use these quotes that I extracted from previous "Featured Committer Profile" in the EF community newsletters dedicated to Jakarta EE so far. But I will also reach out to the community mailing lists to invite the contributors and committers of the WG to dedicate a quote if interested for us to add to this page later.

Please note that for the contributors section at the bottom, the idea is to feature and pull the images of all contributors and committers of the EE4J on that section and they should change when the page refreshes.

### PR DESCRIPTION
**Pull request by ericpoirier**
Fixes #1423 

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by ericpoirier** [comment]
We have the +1 from Shabnam, she'd like to get that merge today.

File: layouts/shortcodes/pages/get-involved/main-content.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                                <li><img class="get-involved-orange-icon" src="/images/get-involved/icons/slack.svg" alt="Slack icon"><a href="https://eclipsefoundationhq.slack.com/?redir=%2Fssb%2Fredirect">Join us on Slack</a></li>
+                            </ul>
+                        </div>
+                        <div class="col-sm-12">
+                            <ul class="list-unstyled">
+                                <li><img class="get-involved-orange-icon" src="/images/get-involved/icons/social-media.svg" alt="Social Media icon"><a href="#social-media">Join us on Social Media</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <ul class="list-unstyled">
```
Comment (autumnfound) [inline layouts/shortcodes/pages/get-involved/main-content.html]: Why do we have an empty list here?

File: layouts/shortcodes/pages/get-involved/testimonials.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+{{ end }}
+
+{{ $testimonials := $scratch.GetSortedMapValues "testimonials" }}
+
+{{ if $testimonials }}
+<div class="get-involved-testimonials testimonials-alternate row">
+    
+        <h2 class="big-text-secondary text-center margin-bottom-40">{{ i18n "home-testimonials" }}</h2>
+        
+            <ul class="solstice-slider owl-carousel owl-theme list-inline" data-slider-lg-count="1" data-slider-md-count="1" data-slider-sm-count="1">
```
Comment (autumnfound) [inline layouts/shortcodes/pages/get-involved/testimonials.html]: Owl injects stuff between the items and the parent container, so the list items aren't semantically correct here. You should just use divs

File: layouts/shortcodes/pages/get-involved/main-content.html (hunk 1 above)
Comment (ericpoirier) [reply layouts/shortcodes/pages/get-involved/main-content.html]: That's a copy paste error.

File: layouts/shortcodes/pages/get-involved/testimonials.html (hunk 2 above)
Comment (ericpoirier) [reply layouts/shortcodes/pages/get-involved/testimonials.html]: I replaced the `<ul>/<li> `tags with `<div>` tags.

**Review by autumnfound — APPROVED** [review]

**Review by chrisguindon — APPROVED** [review]

**Pull request merged**